templates/20-validation-webhook.deployment.yaml.tmpl

```diff
@@ -18,15 +18,15 @@ spec:
     spec:
       serviceAccountName: "#SANAME#"
       initContainers:
-      - image: "#IMG#:#IMAGETAG#"
+      - image: "#IMG#:${IMAGE_TAG}"
```

Comment:
Where does `${IMAGE_TAG}` come from? How does it relate to the `${IMAGETAG}` from the Makefile?

Reply:
IMAGE_TAG comes from the template which saasheader applies.  Defined here:
https://github.com/openshift/managed-cluster-validating-webhooks/blob/master/build/00-osd-managed-cluster-validating-webhooks.selectorsyncset.yaml.tmpl#L4-L5

Reply:
Ahh, I see. Thanks!